docs/reference/ccr/apis/follow/post-resume-follow.asciidoc

```diff
@@ -63,6 +63,15 @@ POST /<follower_index>/_ccr/resume_follow
 ==== Request Body
 include::../follow-request-body.asciidoc[]
 
+==== Authorization
+
+If the {es} {security-features} are enabled, you must have `write` index
+privileges for the follower index and `read` index privileges for the leader
+index. You must have `manage_ccr` and `monitor` cluster privileges on the
```

Comment:
The monitor indices privilege in both clusters is needed instead of monitor cluster privilege.